homeassistant/components/home_connect/__init__.py

```diff
@@ -138,11 +138,18 @@ async def _async_service_program(call, method):
         """Execute calls to services taking a program."""
         program = call.data[ATTR_PROGRAM]
         device_id = call.data[ATTR_DEVICE_ID]
-        options = {
-            ATTR_KEY: call.data.get(ATTR_KEY),
-            ATTR_VALUE: call.data.get(ATTR_VALUE),
-            ATTR_UNIT: call.data.get(ATTR_UNIT),
-        }
+
+        options = []
+
+        option_key = call.data.get(ATTR_KEY)
```

Comment:
The option key is required in the service schema. Please use `dict[key]` to access required dict keys.

Reply:
No it is not. The `SERVICE_PROGRAM_SCHEMA` consists of two sets. One contains the `ATTR_KEY`, the other does not.